test/tests/api/v1_1/__init__.py

```diff
@@ -16,5 +17,6 @@
     'profiles.tests'
     'config.tests',
     'workflowTasks.tests',
-    'workflows.tests'
+    'workflows.tests',
+    'auth.tests'
```

Comment:
this will always run auth.tests as part of the v1.1 group.   Does this still work when config.json disables authentication?